### ISSUE TITLE
Some images, in particular in the blog post previews on the homepage, may not have a text alternative, so people using a screen reader cannot access the information

### ISSUE THREAD

**Issue by sssoz** [oaworks/oa-works#128]
This fails [WCAG 2.1 success criterion 1.1.1: **non-text content**](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html).

**Comment by thinkbulecount2** [comment]
Going to resolve this as we fetch the blog excerpt and image via Ghost and there is no way to fetch `alt` text from the API.

**Comment by sssoz** [comment]
@thinkbulecount2  I believe I might have been wrong and got this confused with another blog API. Have you taken a look at Ghost’s API documentation? https://ghost.org/docs/content-api/#posts Looks like we _can_ display the alt text.

**Comment by thinkbulecount2** [comment]
Ok, will take a look, @sssoz!

**Comment by sssoz** [comment]
Done in https://github.com/oaworks/oa-works/pull/147/commits/c6dee654ecc763fcde35b61e71df2f3121315b8b 
Updated [a11y statement](https://oa.works/policies/accessibility/) accordingly
